test/Dialect/TorchConversion/torch-backend-to-linalg-on-tensors-no-mlprogram.mlir

```diff
@@ -0,0 +1,15 @@
+// RUN: torch-mlir-opt -p 'builtin.module(torch-backend-to-linalg-on-tensors-backend-pipeline{use-mlprogram=0})' -split-input-file %s | FileCheck %s
```

Comment:
Maybe add the same test with `use-mlprogram=1` and a check for the global?

Reply:
The pass that adds the global is tested in `test/Conversion/TorchConversionToMLProgram/basic.mlir`; then the pass that checks the pipeline altogether doesn't check individual components (including this global). Does it make sense to have a test for this individual component?

Reply:
I don't mean to really check the global, but the flag working as expected (kind of what the test setting =0 does). Does that make it more clear?

Reply:
I added a version with that flag being enabled :)